Tests/Functional/BackEnd/AbstractEventMailFormTest.php

```diff
@@ -130,6 +140,28 @@ public function sendEmailSendsEmailWithNameOfRegisteredUserInSalutationMarker()
         self::assertContains('Joe Johnson', $this->mailer->getFirstSentEmail()->getBody());
     }
 
+    /**
+     * @test
+     */
+    public function sendEmailUsesTypo3DefaultFromAddressAsSender()
```

Comment:
We also need a test for the case where the first organizer is used.

Reply:
You're right. That's what `sendEmailUsesFirstOrganizerAsSender()` should do. I'll fix that test and add another one that tests the replyTo address